_posts/en/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,264 @@
+    argument against its spending transaction's nLockTime field in the
+    same way the nLockTime field is compared to the containing
+    block's height or nTime field.  This allows scripts to prevent money
+    received to them from being spent until after a certain future
+    height or time.  However, the Blockstream post explains that this
+    also has the non-obvious implication that it's possible to create a
+    script that's unspendable because it requires the simultaneous use
+    of both heightlocks and timelocks.  For example, a payment to
+    following script can never be spent:
+
+       1 OP_CLTV OP_DROP 500000001 OP_CLTV
+
+    The first condition allows the spending transaction to be included
+    in block 1 or later, so any block after early January 2009, and the
+    second condition allows it to be included in any block after early
+    November 1985.  Both conditions are true today---but they can't both
+    be satisfied using a transaction's single nLockTime field.  The post
+    notes that the same problem can apply when a transaction has
+    multiple inputs each with their own script.  For example:
+
+        Input 0:
+          1 OP_CLTV
+
+        Input 1:
+          500000001 OP_CLTV
+
+    The problem also applies to relative timelocks and relative
+    heightlocks created using [BIP68][] sequence numbers and the
+    [BIP112][] `OP_CHECKSEQUENCEVERIFY` opcode.
+
+    The post notes that the [Miniscript][topic miniscript] compiler has been updated to
+    deal with the possible conflicts as best as possible.  It will
+    identify when one or more of the ways to satisfy a script contains
+    conflicting locks and will return a warning.  Because of the
+    conflict, it'll also not be able to provide full analysis of the
+    script.  Additionally, the compiler for the policy language will now
+    deliberately fail on policies that mix timelocks and heightlocks,
+    e.g.  `thresh(3,after(1),after(500000001),pk(A))`.  Note: as of this
+    writing, this change is only a [pending PR][rust-miniscript #121] to
+    the Rust version of the miniscript library and has not yet
+    propagated to the online live demos for [miniscript][miniscript demo] or [minsc][].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Swan supports sending to bech32 addresses:**
+  Swan, through their custodian Prime Trust, now [supports withdrawals][swan bech32 send]
+  to [bech32][topic bech32] addresses.
+
+- **Ledger adds manual coin selection support:**
+  Moving from a first-in, first-out UTXO selection model, Ledger now also
+  supports [manual coin selection][ledger coin control article] for either privacy
+  or fee-minimization benefits.
+
+- **Sparrow wallet announced:**
+  [Sparrow wallet][sparrow twitter thread] is a new desktop wallet supporting
+  single or multisignature addresses, PSBT, hardware wallets, and [coin
```

Comment:
Either here, in the Joinmarket item, or in both, PSBT could be linked to `topic psbt`.

Reply:
Added to JM section.